requirements.txt

```diff
@@ -20,5 +20,4 @@ parse
 tooz # Apache-2.0
 sherlock>=0.4.1 # MIT
 kubernetes # Apache-2.0
-# email
-# smtplib
+pre-commit
```

Comment:
This shouldn't be here, we should move this to `test-requirements.txt` -- we cannot have Staffeln depend on `pre-commit`.